Lib/pathlib.py

```diff
@@ -562,15 +467,31 @@ def as_uri(self):
         """Return the path as a 'file' URI."""
         if not self.is_absolute():
             raise ValueError("relative path can't be expressed as a file URI")
-        return self._flavour.make_uri(self)
+
+        # FIXME: move this implementation to os.path.fileuri()
```

Comment:
Shouldn't have `FIXME` comments left.

Reply:
In what circumstances can TODOs/FIXMEs/etc be added to the code? Are there none?

Reply:
Maybe when adding a note to existing code that’s not touched by the PR.  But adding a fixme in new code seems strange, the new code should have the fix! 🙂